README.md

```diff
@@ -57,56 +71,60 @@ Currently there are three types of values that can be represented in pringle:
   - see integers
 
 ### Variables
+
 You must declare and assign a variable at the same time. The expression before the `var` function/keyword is the value of the variable and the identifier after `var` is the variable's name. Note that all variables are in the global scope.
-* variable declaration and use example: 
+
 ```
+# variable declaration and use example
 3 4 + var n 
 ```
-*assigns the value 7 to the variable n*
 
 ```
-n print
+# assigns the value 7 to the variable n
```

Comment:
It would be more clear if this comment was in the previous code block with `3 4 + var n`